### DISCUSSION TITLE
Change Auto Save location

### DISCUSSION THREAD

**Discussion by martin-kirilov** [coteditor/CotEditor#1304] [Q&A]
Hi, I want to change the location where files are autosaved (cached), since by default it is iCloud Drive, but I don't have enough free space on iCloud and I'm constantly getting warnings about it. Do you have any suggestions?

**Comment by 1024jp** [comment]
There is currently no way to change the location of the Autosave. CotEditor uses the standard APIs by Apple (namely, NSDocument) for handing document files and it uses iCloud as long as the user enables it.

One possibilisty is disabling the iCloud document feature only for CotEditor in the System Preferences. Then CotEditor changes the Autosave location to the local. It also disables the feature having CotEditor directory in iCloud but it can not important for you (you can still save your documents somewhereelse in iCloud).

![Screen Shot 2022-05-23 at 9 42 52](https://user-images.githubusercontent.com/1165044/169724180-8e98b31f-4a5a-4232-ae8c-13f52ed986b2.png)

**Comment by martin-kirilov** [comment]
When I did that, it stopped autosaving when I quit CotEditor. Instead, it prompts me for each open tab to select a save location. Could you clarify a bit more your solution?
